lib/trento/application/integration/discovery/payloads/cloud_discovery_payload.ex

```diff
@@ -0,0 +1,83 @@
+defmodule Trento.Integration.Discovery.CloudDiscoveryPayload do
+  @moduledoc """
+  Cloud discovery integration event payload
+  """
+
+  @required_fields []
```

Comment:
Wouldn't be the `provider` field required here?

Reply:
Not really since we default to `:unknown` :sweat_smile: 

```
iex(4)> %{"metadata" => %{"compute" => %{"name" => "alessio"}}} |> Trento.Integration.Discovery.CloudDis
coveryPayload.new
{:ok,
 %Trento.Integration.Discovery.CloudDiscoveryPayload{
   metadata: nil,
   provider: :unknown
 }}
```